src/components/summary-list/index.md.njk

```diff
@@ -30,24 +30,31 @@ Do not use it for tabular data or a simple list of information or tasks, like a
 
 There are 2 ways to use the summary list component. You can use HTML or, if you’re using [Nunjucks](https://mozilla.github.io/nunjucks/) or the [GOV.UK Prototype Kit](https://govuk-prototype-kit.herokuapp.com), you can use the Nunjucks macro.
 
-### Summary list with actions
+{{ example({group: "components", item: "summary-list", example: "without-actions", html: true, nunjucks: true, open: false}) }}
+
+###  Adding actions to each row
 
 You can add actions to a summary list, like a ‘Change’ link to let users go back and edit their answer.
 
 For sighted users, the actions get their context from the other content in the row they appear in.
 
 Assistive technology users, like those who use a screen reader, may hear the links out of context and not know what they do.
+
 To give more context, add visually hidden text to the links. This means a screen reader user will hear a meaningful action, like ‘Change name’ or ‘Change date of birth’.
 
 {{ example({group: "components", item: "summary-list", example: "default", html: true, nunjucks: true, open: false, titleSuffix: "second"}) }}
 
-### Summary list without actions
+### Adding actions to only some rows
 
-{{ example({group: "components", item: "summary-list", example: "without-actions", html: true, nunjucks: true, open: false}) }}
+If you have a mix of rows with and without actions, add the `govuk-summary-list__row--no-actions` modifier class to the rows that do not have actions.
+
+{{ example({group: "components", item: "summary-list", example: "mixed-actions", html: true, nunjucks: true, open: false}) }}
+
+### Removing the borders
 
-### Summary list without borders
+The summary list includes separating borders to help users read each row and its action.
 
-If you do not include actions in your summary list and it would be better for your design to remove the separating borders, use the `govuk-summary-list--no-border` class.
+If your summary list does not have any actions, you can choose to remove the separating borders with the `govuk-summary-list--no-border` class.
 
 {{ example({group: "components", item: "summary-list", example: "without-borders", html: true, nunjucks: true, open: false}) }}
 
```

Comment:
Not a big deal, but I'm not sure I understand when someone would choose to selectively remove borders for rows... 
- To show if a row has an action or not? Seems like it'd look a bit off
- To group rows together? That'd only be a visual-only distinction which would not met WCAG

Reply:
Probably the most common is to remove the bottom border from the very last row – we even have [an issue to make that the default behaviour](https://github.com/alphagov/govuk-frontend/issues/1909).

Not sure we know enough about other use cases to go into detail really. Unsure about whether grouping being visual only would be a WCAG fail… likely depends how important the grouping is, and whether it is communicated another way e.g. through the labels used.

For example, this sort of thing would probably be OK…

```
Home address           1234 Lower Avenue           Change
Home phone number      01234 567890                Change
---------------------------------------------------------
Work address           21 High Street              Change
Work phone number      01234 098765                Change
---------------------------------------------------------
```

Reply:
Ahh okay, thanks for explaining. They'd be grouped in that instance (as similar/repeated) field names.